pkg/server/handlers/metrics.go

```diff
@@ -56,36 +56,57 @@ func (m metrics) initShootMetrics() {
 	metricShootState := prometheus.NewGaugeVec(prometheus.GaugeOpts{
 		Name: "garden_shoot_state",
 		Help: "State of a Shoot cluster",
-	}, []string{"name", "project", "cloud", "version", "region", "seed", "createdAt"})
+	}, []string{"name", "project", "cloud", "version", "region", "seed", "operation", "is_seed", "mail_to"})
 
 	metricShootNodeCount := prometheus.NewGaugeVec(prometheus.GaugeOpts{
 		Name: "garden_shoot_nodes_count",
 		Help: "Node count of a Shoot cluster",
 	}, []string{"name", "project"})
 
+	metricShootStateConditions := prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Name: "garden_shoot_state_conditions",
+		Help: "Conditions of a Shoot cluster",
+	}, []string{"name", "project", "condition", "status", "operation", "mail_to"})
+
 	prometheus.Register(metricShootState)
 	prometheus.Register(metricShootNodeCount)
+	prometheus.Register(metricShootStateConditions)
 
 	m.collect(func() {
-		var state float64
 		shoots, err := m.k8sGardenClient.GardenClientset().GardenV1beta1().Shoots(metav1.NamespaceAll).List(metav1.ListOptions{})
 		if err != nil {
 			logger.Logger.Info("Unable to fetch shoots. skip shoot metric set...")
 			return
 		}
 
 		for _, shoot := range shoots.Items {
-			state = 4 // unknown
+			var (
+				mailTo         string
+				nodeCount      int
+				operationState float64 = 4 // unknown
+				operationType          = "Unknown"
+				isSeed                 = "False"
+			)
+
+			if shoot.Annotations != nil {
+				mailTo = shoot.Annotations[common.GardenOperatedBy]
+				if value, ok := shoot.Annotations[common.ShootUseAsSeed]; ok && value == "true" {
```

Comment:
`strconv.ParseBool(value)` to convert string -> bool

Reply:
But then we can't  write this as one liner ``strconv.ParseBool()`` returns ``(bool, error)`` and we need an error check...

Reply:
actually you can only use

```go
value ,_ := strconv.ParseBool(shoot.Annotations[common.ShootUseAsSeed])
```

the error is not needed and you don't care for it. It is only thrown It can't be converted / its empty